apps/rule-manager/conf/evolutions/default/12.sql

```diff
@@ -0,0 +1,16 @@
+-- !Ups
+
+ALTER TABLE rules_draft
+    ADD COLUMN rule_order INT UNIQUE;
```

Comment:
The `rule_order` column is initially added without a `NOT NULL` constraint, as including this would require setting a default value for the existing rows, which would violate the `UNIQUE` constraint. After populating the column by copying the `id` value, we add the `NOT NULL` constraint.